project_singlefamilydetached/housing_characteristics/Insulation Finished Roof.tsv

```diff
@@ -1,2 +1,2 @@
-Option=Uninsulated	Option=R-30
-0.5	0.5
```

Comment:
Were we previously simulating any SFD houses with finished roofs? Anyhow, seems like you don't need the `Dependency=Geometry Building Type` column here since there's only one enumeration in the list.

Reply:
This TSV was added after the national SFD project was over; but since the mf project this has been the case and I was directed to add the dependency and set-it up as "None" for all SFs. I guess we can remove the TSV all together from this project?

Reply:
We can't remove "Insulation Finished Roof.tsv" from project_singlefamilydetached (see https://trello.com/c/N5xhoBQL/126-why-is-insulation-finished-rooftsv-in-all-projects) unless we do something with "Roof Material.tsv" (e.g., split into separate "Unfinished Roof Material.tsv" and "Finished Roof Material.tsv"). My point was just that you don't need column A (`Dependency=Geometry Building Type`) in this file.

Reply:
As long as `Insulation Finished Roof.tsv` is in the project, then it will get sampled and show up in `results.csv`, right (even if it isn't used)? Adding `Dependency=Geometry Building Type` will at least allow us to set `Insulation Finished Roof` to `None` for SFD homes, so it seems like a good idea to me.

Reply:
There's only one option (None) set at 100%, so the dependency has no impact. You'll always get None in the results.csv with or without the dependency.

Reply:
So what's the direction for going ahead here? remove the dependency or keep it?

Reply:
Remove the dependency. It's useless.